### PR TITLE
Fix: Fix v4.4.0 bug 0904

### PR DESCRIPTION
**Pull request by wen-qing-255**

### CODE REVIEW

File: src/app/ui/components/Input/NumberInput.jsx [hunk 1]
```diff
@@ -41,6 +41,7 @@ const NumberInput = ({
     function onAfterChangeWrapper(changedValue) {
         let numericValue = parseFloat(changedValue);
         let useEdgeValue = false;
+        console.log({ numericValue });
```
Comment (jane-rose) [inline src/app/ui/components/Input/NumberInput.jsx:44]: log

File: src/app/ui/widgets/PrintingVisualizer/Visualizer.jsx [hunk 2]
```diff
@@ -605,12 +607,18 @@ class Visualizer extends PureComponent {
                             <Input
                                 size="small"
                                 placeholder="0"
+                                key={this.randomKey}
```
Comment (jane-rose) [inline src/app/ui/widgets/PrintingVisualizer/Visualizer.jsx:610]: don't just use number for key

File: src/server/services/socket/socket-http.ts [hunk 3]
```diff
@@ -394,9 +395,9 @@ class SocketHttp {
                     type: CONNECTION_TYPE_WIFI
                 });
             } else {
-                this.socket && this.socket.emit('sender:status', {
-                    data: this.getGcodePrintingInfo(state)
-                });
+                // this.socket && this.socket.emit('sender:status', {
```
Comment (jane-rose) [inline src/server/services/socket/socket-http.ts:398]: check this. 'sender:status' event used on `WorkspaceVisualizer/Visualizer`

File: src/app/models/ModelGroup.ts [hunk 4]
```diff
@@ -214,7 +214,7 @@ class ModelGroup extends EventEmitter {
      * Note: for performance consideration, don't call this method in render.
      */
     public getBoundingBox() {
-        return ThreeUtils.computeBoundingBox(this.object) as Box3;
```
[the remark this answers is not in the document]
Comment (wen-qing-255) [reply src/app/models/ModelGroup.ts:217]: with Box3 Type, never get the correct boundingbox

File: src/server/services/socket/socket-http.ts (hunk 3 above)
Comment (wen-qing-255) [reply src/server/services/socket/socket-http.ts:398]: `sender:status` event used to update workingProgress data. Now use `Marlin:state` to update.

**Pull request merged**